pkgs/tools/networking/isync/default.nix

```diff
@@ -15,6 +15,9 @@ stdenv.mkDerivation rec {
   patches = [
     # Fixes "Fatal: buffer too small" error
     ./0001-Increase-imap_vprintf-buffer-size.patch
+    # Fix #202595: SSL error "Socket error: ... unexpected eof while reading"
+    # Source: https://sourceforge.net/p/isync/isync/ci/b6c36624f04cd388873785c0631df3f2f9ac4bf0/
+    ./work-around-unexpected-EOF-error-messages-at-end-of-SSL-connections.patch
```

Comment:
I would've preferred to fetch this patch directly from SourceForge but apparently that's not possible yet... :o
https://sourceforge.net/p/forge/feature-requests/614/